test/IECoreGL/Camera.py

```diff
@@ -114,13 +114,13 @@ def testXYOrientation( self ) :
 		dimensions = i.dataWindow.size() + imath.V2i( 1 )
 		index = dimensions.x * dimensions.y//2 + dimensions.x - 1
 		self.assertEqual( i["A"][index], 1 )
-		self.assertEqual( i["R"][index], 1 )
+		self.assertAlmostEqual( i["R"][index], 1, 6 )
```

Comment:
Do you know why we're suddenly getting non-exact results here, and in the other GL tests?

Reply:
I do not unfortunately, do you have any intuition?